containers/frontend/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM northwestwitch/cgbeacon2:latest
```

Comment:
I'm hosting the image on my docker hub now because I don't have access to the clinicalgenomics one, but I plant to move the images there